benchmark/parseandstatcompetition.cpp

```diff
@@ -154,11 +155,11 @@ static void GenStatPlus(Stat &stat, const dom::element &v) {
     break;
   case dom::element_type::STRING: {
     stat.stringCount++;
-    std::string_view sv = v.get<std::string_view>();
+    auto sv = std::string_view(v);
```

Comment:
The switch-case + cast is nice.

Reply:
The cast still requires exceptions, which bothers me. I am just not sure how to make it better.

I just realized it could have been `std::string_view sv = v;` which has better optics IMO. Gonna change that.